src/accountant/core.cljs

```diff
@@ -9,7 +9,22 @@
            goog.history.Html5History
            goog.Uri))
 
-(defonce history (Html5History.))
+(defn- transformer-create-url
+  "Replacement createUrl TokenTransformer that avoids preserving the query string,
+  as Google's version incorrectly does. (See https://goo.gl/xwgUos)"
```

Comment:
I didn't check the link. Kept it from the previous version of the code. Don't know where it should link. I could remove it, I guess...